### PR TITLE
updated README with corrections and suggestions

### PR DESCRIPTION
**Pull request by natkuhn**
Amy, thank you so much for doing this! There were still some corrections needed in section 8. I added a new section 9 for instructions on how to actually use it, which is not so easy if you want to use two devices on your local network. I also made some minor changes to better reflect the exact steps you take.

I have not looked at the "solution" branch, but my guess is that it would be good to recreate it based on following these steps, so they match up. LMK if you'd like me to do that.

I also noticed as I was doing this that I don't think the formatting after "This function does two main things" came out the way you intended.

Thanks again, please let me know if there is anything more I can do.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,14 +88,15 @@ Before starting this codelab, make sure that you've installed:
     1. Open your app at http://localhost:5000
         You should see your copy of FirebaseRTC which has been connected to your Firebase project.
     The app has automatically connected to your Firebase project.
+    > Note: if you want to access the server from other devices on your local network, you need to add `-o 0.0.0.0` at the end of the `firebase serve` command.
```
Comment (meldaravaniel) [inline README.md:91]: oh nice! i hadn't gotten as far as playing with this.

File: README.md [hunk 2]
```diff
@@ -88,14 +88,15 @@ Before starting this codelab, make sure that you've installed:
...
     
     In this application, each video chat session is called a __room__. A user can create a new room by clicking a button in their application. This will generate an ID that the remote party can use to join the same room. The ID is used as the key in Cloud Firestore for each room.
     
     Each room will contain the `RTCSessionDescriptions` for both the offer and the answer, as well as two separate collections with [ICE candidates](https://webrtcglossary.com/ice/#:~:text=ICE%20stands%20for%20Interactive%20Connectivity,NAT%20traversal%20used%20in%20WebRTC.) from each party.
 
-    Your first task is to implement the missing code for creating a new room with the initial offer from the caller. Open public/app.js and find the comment `// Add code for creating a room here` and add the following code:
+    Your first task is to implement the missing code for creating a new room with the initial offer from the caller. Open public/app.js, find the comment `// Add code for creating a room below`, and add the following code. (You should add the code before the corresponding `// Add code for creating a room above`; the same goes for the rest of the code blocks.)
```
Comment (meldaravaniel) [inline README.md:99]: tbh I didn't really like the syntax of having the double lines, but I like this clarification.  It seemed obnoxious to have to type that multiple times.  Good compromise.

File: README.md [hunk 3]
```diff
@@ -159,18 +160,18 @@ Before starting this codelab, make sure that you've installed:
                                         localName, remoteName) {
         const candidatesCollection = roomRef.collection(localName);
 
-        peerConnection.addEventListener('icecandidate', event -> {
+        peerConnection.addEventListener('icecandidate', event => {
```
Comment (meldaravaniel) [inline README.md:163]: ah, thanks.  I did get it right in my version of the solution branch, but being primarily java-based myself, this doesn't set my alarms off as readily.  Dunno why they chose skinny arrows for java... *grumble*

File: README.md [hunk 4]
```diff
@@ -159,18 +160,18 @@ Before starting this codelab, make sure that you've installed:
...
         });
 
-        roomRef.collection(remoteName).onSnapshot(snapshot -> {
-            snapshot.docChanges().forEach(change -> {
+        roomRef.collection(remoteName).onSnapshot(snapshot => {
+            snapshot.docChanges().forEach(change => {
                 if (change.type === "added") {
                     const candidate = new RTCIceCandidate(change.doc.data());
-                    peerConneciton.addIceCandidate(candidate);
+                    peerConnection.addIceCandidate(candidate);
```
Comment (meldaravaniel) [inline README.md:174]: also correct in solution.

File: README.md [hunk 5]
```diff
@@ -181,7 +182,23 @@ Before starting this codelab, make sure that you've installed:
         * listens for added ICE candidates from the remote peer and adds them to its `RTCPeerConnection` instance. 
     It is important when listening to database changes to filter out anything that isn't a new addition, since we otherwise would have added the same set of ICE candidates over and over again.
     
-    Complete this step by uncommenting the calls to this function in both the `joinRoomById` and `createRoom` methods.
+    Complete this step by uncommenting the calls to this function in both the `joinRoomById` and `createRoom` methods; you can find them after `// Uncomment to collect ICE candidates below`.
+
+1. __Try it out__
+
+    1. Open a new browswer tab at http://localhost:5000, or refresh the tab you opened above.
+    1. Mute your speakers to avoid loud piercing feedback!
```
Comment (meldaravaniel) [inline README.md:190]: bahaha! so true.  I had this on in the living room and forgot that would happen.  It was....shocking. 😝

File: README.md [hunk 6]
```diff
@@ -181,7 +182,23 @@ Before starting this codelab, make sure that you've installed:
...
+    1. Click on __Open camera & microphone__; give permission to the app to use them, if requested.
+    1. Click on __Create room__. The app will display the room ID.
+    1. Open another browswer tab, click __Open camera & microphone__, and click __Join room__. Paste in the ID from the previous step. The two instances should connect.
+    
+    To connect with a different device on your local network:
```
Comment (meldaravaniel) [inline README.md:195]: Super useful!

**Review by meldaravaniel — APPROVED** [review]
Thanks so much for fixing the bits I missed and enhancing it further.  Out of date documentation is one of my pet peeves.  😓

**Comment by meldaravaniel** [comment]
> I also noticed as I was doing this that I don't think the formatting after "This function does two main things" came out the way you intended.

Ah, you're right.  I can fix that pup.

**Comment by natkuhn** [comment]
Hey, thanks for doing that! I was fun watching the emails roll in as you worked your way through it. Only my second PR ever, and the last one was long enough ago that I forgot everything about it.

**Comment by meldaravaniel** [comment]
The pleasure is mine!  I like to try to make PRs an enjoyable and educational experience at work, so may as well carry that over into the 'real world', otherwise no on wants to pitch in.

**Pull request merged**